src/hooks/v2/ProjectMetadata.ts

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios'
+
+import { consolidateMetadata } from 'models/project-metadata'
+import { useQuery } from 'react-query'
+import { ipfsCidUrl } from 'utils/ipfs'
+
+export default function useV2ProjectMetadata(metadataCID?: string) {
```

Comment:
do we need a separate hook from the one used in v1? if so, should we move the current /hooks/ProjectMetadata.ts to hooks/v1/ ?

Reply:
Probably not I think! Great catch 🙏